### PR TITLE
promises for upload

### PR DESCRIPTION
**Pull request by tomxhu**

### CODE REVIEW

File: api/entries.js [hunk 1]
```diff
@@ -48,13 +51,16 @@ module.exports = {
     	var data = JSON.parse(req.body.entries);
 
         console.log("\nMulti-Post Request");
-        console.log(req.body.entries);
+        // console.log(req.body.entries);
+
+        var promises = []
```
Comment (alexjohnson505) [inline api/entries.js:56]: Love it!

**Comment by alexjohnson505** [comment]
:+1: 

Does this replace https://github.com/spacial-player-data-visualization/g-player-visualization/pull/39 ?

**Comment by tomxhu** [comment]
yes this is the replacement

**Comment by alexjohnson505** [comment]
Awesome! https://github.com/spacial-player-data-visualization/g-player-visualization/pull/39 deleted. Looks good to me.

**Pull request merged**